### PR TITLE
Neighbors fix

### PR DESCRIPTION
**Pull request by tchangalov**

### CODE REVIEW

File: virotour_local/flask/virotour/integration_tests/test_basic_integration.py
Comment (tpbnick) [inline virotour_local/flask/virotour/integration_tests/test_basic_integration.py:10]: Will this need to be re-commented out?

**Review by tpbnick — COMMENTED** [review]
`test_basic_integration` is passing for me:
image

Changes don't look like they will impact anything on our end.  I made one comment.

@nguyendviet I know you wanted to not merge anything in today.  If you have time can you check?

**Review by nhomsombath — APPROVED** [review]

**Comment by nguyendviet** [comment]
@tpbnick @tchangalov I checked and this branch contains old Flutter code so the list tour doesn't work on my end. Feel free to demo this branch, just don't merge to `development` today.

One thing you can do in the future is to rebase or merge the target branch (e.g. `development`) to your feature branch first. That way, you can resolve conflicts before merging or creating a PR.

**Review by nguyendviet — APPROVED** [review]

**Pull request merged**
